AndroidSDKCore/src/main/java/com/leanplum/internal/RequestSequence.java

```diff
@@ -0,0 +1,18 @@
+package com.leanplum.internal;
+
+/**
+ * Public interface for during the call sequence of database
+ * read/write operations.
+ *
+ * @author sayaan
```

Comment:
javadoc: `@author` tags are not needed. we can see history in git.

Reply:
done